var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -117,8 +136,9 @@ class Opencv(CMakePackage, CudaPackage):
     depends_on('vtk', when='+vtk')
     depends_on('qt', when='+qt')
     depends_on('java', when='+java')
+    depends_on('ant', when='+java')
```

Comment:
Is this only needed at build time?

Reply:
yes, I believe so. will fix.

Reply:
May I ask what if I want to turn off java support for opencv? How can specify it when `spack install`?

Reply:
You can use:
```console
$ spack install opencv~java
```
You can also set this default in your `packages.yaml` so you don't have to specify it each time.

Reply:
thx for the information!